### DISCUSSION TITLE
Notifications And Reminders

### DISCUSSION THREAD

**Discussion by ElTimuro** [documenso/documenso#860] [Ideas]
We are thinking about the notification infrastructure we want to add to Documenso. As this defines a lot about how the product feels, we wanted to hear your thoughts.

> It will be some time until we action this, but we like to think about things early

- What kinds of notification channels would you expect/ prefer: Email, Notification "Bell" in Login, SMS, Push (Desktop or Mobile), etc.
- What notifications are actually interesting? Reminders? Summaries? Team Actions? None? Realtime "everything"

Let us know what you think 🔔🤔🔕

**Comment by zepinex** [comment]
That would be a very good feature to implement.
